__tests__/snapshotTests.js

```diff
@@ -1,4 +1,4 @@
-/* Component tests generated by react-snapshot-test-generator on 2016-11-03T18:34:51.359Z */
+/* Component tests generated by react-snapshot-test-generator on 2016-11-14T14:51:37.022Z */
```

Comment:
If you want, you can apply this #159 on your branch, this will remove timestamp bumping. This will only occur when we manually update snapshots.